quay-tectonic-identity.md

```diff
@@ -0,0 +1,72 @@
+# Quay Enterprise OIDC Auth with Tectonic Identity 
+
+## Export certificate information from cluster for TLS verification of Tectonic Identity: 
+
+```
+kubectl -n tectonic-system get secret tectonic-ca-cert-secret -o json | jq -r '.data[]' | base64 -d >> tectonic-ca-ingress.crt
+kubectl -n tectonic-system get secret tectonic-ingress-tls-secret -o json | jq -r '.data["tls.crt"]' | base64 -d >> tectonic-ca-ingress.crt
+```
+## Upload Cert to QE
+
+Upload the `tectonic-ca-ingress.crt` certificate to QE via the superuser panel or follow the [documentation](https://coreos.com/quay-enterprise/docs/latest/insert-custom-cert.html) upload manually. 
+
+## Configure Tectonic Identity
+
+Edit the tectonic-identity configmap: 
+
+```
+kubectl -n tectonic-system edit cm tectonic-identity
+```
+
+Add an client definition under the `staticClients` section for Quay Enterprise: 
+
+```
+  - id: quay-enterprise
+    name: Quay Enterprise
+    secret: rEpLaCeThIs
+    redirectURIs:
+      - ''
+```
+
+`secret:` is a user generated string that will later be passed to Quay Enterprise. Use `date | md5sum` or `openssl rand -base64 15` to generate a random string. 
+`redirectURIs:` URLs that must be passed to the tectonic-identity configmap. These are provided by Quay Enterprise in the next step.
```

Comment:
`redirectURIs`:

Reply:
and is this the right sequence? any reason not to grab the redirectURIs first?

Reply:
it is URI and yes the sequencing is meant to provide the most likely chance at success on first configuration by ensuring the user can copy and paste the redirect uri's from the superuser panel